conceptual/EFCore.PG/mapping/enum.md

```diff
@@ -18,12 +18,23 @@ This causes the EF Core provider to create your enum type, `mood`, with two labe
 If you are using `context.Database.Migrate()` to create your enums, you need to instruct Npgsql to reload all types after applying your migrations:
 
 ```c#
-context.Database.Migrate();
+await context.Database.MigrateAsync(token);
 
-using (var conn = (NpgsqlConnection)context.Database.GetDbConnection())
+if (context.Database.GetDbConnection() is NpgsqlConnection npgsqlConnection)
 {
-    conn.Open();
-    conn.ReloadTypes();
+    if (npgsqlConnection.State is ConnectionState.Closed or ConnectionState.Broken)
```

Comment:
This part shouldn't be needed - in this context, GetDbConnection should always return a closed connection - or are you seeing something else?

Reply:
No, I don't. I'm just not sure I'll always get closed connection here, so just not to get an exception that connection is already opened.

Reply:
The connection should be closed, so I think it's OK for the docs to not show this check.